src/platform/datapath_winkernel.c

```diff
@@ -600,24 +600,9 @@ CxPlatDataPathQuerySockoptSupport(
 
     } while (FALSE);
 
-    do {
-        RTL_OSVERSIONINFOW osInfo;
-        RtlZeroMemory(&osInfo, sizeof(osInfo));
-        osInfo.dwOSVersionInfoSize = sizeof(osInfo);
-        NTSTATUS status = RtlGetVersion(&osInfo);
-        if (NT_SUCCESS(status)) {
-            DWORD BuildNumber = osInfo.dwBuildNumber;
-            //
-            // Some USO/URO bug blocks TTL feature support on Windows Server 2022.
```

Comment:
Same remark here for the comment.

Reply:
Agreed, I will add back those comments in the next iteration.